src/doctests.jl

```diff
@@ -490,21 +490,23 @@ function fix_doctest(result::Result, str, doc::Documenter.Document; prefix::Muta
     r = Regex(rcode)
     codeidx = findfirst(r, content)
     if codeidx === nothing
-        @warn "could not find code block in source file"
+        @warn "could not find code block in source file $filename"
```

Comment:
Actually it would be kinda helpful if one would see the code block. Any reason why I shouldn't just print it here?

Reply:
I think that would be fine. I think we do that for some other messages already.